src/CONST.ts

```diff
@@ -652,6 +652,7 @@ const CONST = {
             MUTE: 'mute',
             DAILY: 'daily',
             ALWAYS: 'always',
+            // 'hidden' is a special report level preference that appears when you look at a thread without commenting on it
```

Comment:
There are actually a few reasons why the hidden preference could be set.

Reply:
@marcaaron Any suggestion on what this comment could be then?

Reply:
I would just remove it. It's better to say nothing than be inaccurate.

Reply:
Removed.

Reply:
It's a special value (next to 3 other that directly correspond to dropdown entries). I believe that an important reason for the regression was that we didn't really understand the contract related to this special value.

But I guess we can leave this trap for future generations 😜

Reply:
Fair, but the reality is that not everything is perfectly documented and values like this are used in different cases (so it might be hard to keep this comment up to date). We've been discussing internally some ways to make this easier and bridge the gap between people working on the frontend without access to the backend, but not quite there yet. For now, if you are not sure about something feel free to ask the engineer assigned to the issue or in Slack.

This is how the feature currently works:

- Hidden preference can be set for a variety of reasons
- Chats with hidden preference do not show in the LHN and should not be counted as "unread" 
- They are hidden from the LHN in the render and are not filtered from the "options list"
- This is because they should still show up in Search, but not the LHN